ckan/templates-bs2/403.html

```diff
@@ -0,0 +1,21 @@
+{% extends "page.html" %}
```

Comment:
Can we render the current `error_document_template.html`? See comment about capturing all errors